BranchMonsterFactory/ViewControllers/MonsterViewerViewController.m

```diff
@@ -35,7 +39,6 @@ @interface MonsterViewerViewController () <MFMailComposeViewControllerDelegate,
 @property (weak, nonatomic) IBOutlet UIButton *cmdTwitter;
 @property (weak, nonatomic) IBOutlet UIButton *cmdFacebook;
 
-@property (weak, nonatomic) IBOutlet UITextView *etxtUrl;
```

Comment:
so we pulled this completely? I think we still want to show a link in the app, in addition to the "Copy Share URL" button. That way people can see the link as soon as it's generated :)